extensions-core/kubernetes-extensions/pom.xml

```diff
@@ -137,6 +138,19 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
+        <plugins>
+          <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <version>3.1.1</version>
+              <configuration>
+              <!-- analyze incorrectly flags this dependency -->
+                <ignoredDependencies>io.kubernetes:client-java-api-fluent:jar:19.0.0</ignoredDependencies>
```

Comment:
what do we mean by "flag"?

Reply:
flags as raises the issues with the dependency: 
when the dependency is explicitly added to <dependencies> section then analyze plugin reports it unused, when the dependency is missing from <dependencies> the analyze plugin reports it missing. 
see https://github.com/apache/druid/pull/15407/commits/f0c4ced3e64060791531789292b00e12cfb47362
in https://github.com/apache/druid/pull/15407
I tried it both ways, upgraded the analyze plugin, unfortunately  in the end the resolution was to ignore the checker alert.